### PR TITLE
Version 0.2.0 Release

### PR DESCRIPTION
**Pull request by daikitag**
Update the changelog for version0.2.0 release

### CODE REVIEW

**Comment by daikitag** [comment]
@jeromekelleher Would it be possible for you to check the changelog for me? If it looks good, I will update the date of the version 0.2.0 in this document, and I hope that we can release the new version soon.

**Comment by daikitag** [comment]
Sorry, it is version 0.1.0, not 0.2.0. However, I think we can actually make the new release as version 1.0.0, as we had validated the proporties of tstrait, many new functionalities are added to implement the simulation frameworks of other external simulators, and I feel that this is the complete package now. Do you have any thoughts on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+- Documentation for `random_sign` input in trait distribution models [#122](https://github.com/tskit-dev/tstrait/pull/122)
+- Modify phrasing in documentation [#123](https://github.com/tskit-dev/tstrait/pull/123)
+- Documentation for specifying causal site IDs [#126](https://github.com/tskit-dev/tstrait/pull/126)
+- Documentation for modifying the numericalization of genotypes [#133](https://github.com/tskit-dev/tstrait/pull/133)
+- Modify the frequency dependence explanation in the documentation [#141](https://github.com/tskit-dev/tstrait/pull/141)
+- Fix typo in documentation [#142](https://github.com/tskit-dev/tstrait/pull/142)
+
+### Contributors:
+
+- Jerome Kelleher
```
Comment (jeromekelleher) [inline CHANGELOG.md:50]: Order these alphabetically?

**Review by jeromekelleher — COMMENTED** [review]
Looks good. I don't think listing out the URLs is an improvement, and you need to be a bit clearer about the breaking status.

I think version 1.0.0 is quite premature (consider how many versions msprime went through before 1.0)

**Comment by daikitag** [comment]
I fixed it now, so would it be possible for you to check it again @jeromekelleher ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,14 +91,14 @@ Test release of the package before releasing it to conda-forge
...
-- Hide private functions and classes {pr}`73`
-- Modify docstring explanations and examples {pr}`76`
+- Hide private functions and classes [#73](https://github.com/tskit-dev/tstrait/pull/73)
+- Modify docstring explanations and examples [#76](https://github.com/tskit-dev/tstrait/pull/76)
 
 ### Documentation:
 
-- Create infrastructure for documentation {pr}`77`
-- Build initial documentation {pr}`78`
-- Add Changelog to documentation {pr}`79`
```
[the remark this answers is not in the document]
Comment (daikitag) [reply CHANGELOG.md]: I looked up the tstrait website, and it seems that {pr} is working, so I modified it just now.

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**